pytorch_blade/src/ltc/disc_backend/backend_impl.cpp

```diff
@@ -246,6 +248,17 @@ std::vector<torch::lazy::BackendDataPtr> DISCBackendImpl::ExecuteComputation(
         arguments, device, default_device_is_cuda);
   }
 
+  auto dump_path =
+      torch::blade::env::ReadStringFromEnvVar("TORCH_DISC_REPLAY_PATH", "");
+  if (!dump_path.empty()) {
+    auto disc_hash_str = torch::lazy::HashToString(disc_hash);
+    TORCH_CHECK(
+        !mkdir(dump_path.c_str(), 0755), "unable to create dir: " + path);
```

Comment:
Need to handle existed path and nonexistent path prefix?

nit: BTW, I'd prefer just print warning log and skip dumping if `mkdir` fails.